qcfractal/queue/executor_adapter.py

```diff
@@ -78,6 +82,10 @@ def _submit_task(self, task_spec: Dict[str, Any]) -> Tuple[Hashable, Any]:
             func, *task_spec["spec"]["args"], **task_spec["spec"]["kwargs"], resources={"process": 1})
         return task_spec["id"], task
 
+    def count_running_workers(self) -> int:
```

Comment:
We should be able to call [this](https://github.com/dask/dask-jobqueue/blob/master/dask_jobqueue/core.py#L485) function.